core/config/engine.cpp

```diff
@@ -36,6 +36,7 @@
 #include "core/license.gen.h"
 #include "core/variant/typed_array.h"
 #include "core/version.h"
+#include "main/main.h"
```

Comment:
This shouldn't reference `main.h` I'd say, each section should be as self-contained as possible, there's only one other case of including `main.h` in `core`. I'd say this should be handled by `Engine` and `main.h` should get its data from there instead (as is done in most other cases, like controlling printing)

Reply:
Sounds good.  I'll go change to that

Reply:
There we go! (I think).  Is this what you were thinking @AThousandShips?

Reply:
Will take a look!